hack/update-pkg-manifest.sh

```diff
@@ -68,7 +68,7 @@ rm -rf ${DEPLOY_PATH}/${RH_BUNDLE_VERSION}
 "$kstomize" build config/manifests | "$opsdk" generate packagemanifests -q --version ${RH_BUNDLE_VERSION} ${RH_PKG_MAN_OPTS} --output-dir ${DEPLOY_PATH} --input-dir ${DEPLOY_PATH} --verbose
 # cat ${DEPLOY_PATH}/${RH_BUNDLE_VERSION}/cockroach-operator.clusterserviceversion.yaml | sed -e "s+RH_COCKROACH_OP_IMAGE_PLACEHOLDER+${RH_COCKROACH_OP_IMG}+g" -e "s+RH_COCKROACH_DB_IMAGE_PLACEHOLDER+${RH_COCKROACH_DATABASE_IMAGE}+g" -e "s+CREATED_AT_PLACEHOLDER+"$(date +"%FT%H:%M:%SZ")"+g"> ${DEPLOY_PATH}/${RH_BUNDLE_VERSION}/csv.yaml
 cat ${DEPLOY_PATH}/${RH_BUNDLE_VERSION}/cockroach-operator.clusterserviceversion.yaml | sed "s+RH_COCKROACH_OP_IMAGE_PLACEHOLDER+${RH_COCKROACH_OP_IMG}+g; s+CREATED_AT_PLACEHOLDER+"$(date +"%FT%H:%M:%SZ")"+g"> ${DEPLOY_PATH}/${RH_BUNDLE_VERSION}/csv.yaml
-for v in $(grep -v '#' ${REPO_ROOT}/crdb-versions.txt)
+for v in $("$faq" -r '.CrdbVersions' "${REPO_ROOT}/crdb-versions.yaml" | cut -d ' ' -f2)
```

Comment:
Could this be done in go too somewhere? Maybe a `hack/openshift` package? Can we get rid of the logic in the shell scripts? I'm not sure if it makes sense but might be easier to maintain?

Reply:
Sure! Let's do that as a follow up maybe? I actually filed https://github.com/cockroachdb/cockroach-operator/issues/549 for that particular reason. At least with this PR we are not regressing in this particular case. What do you think?

Reply:
That's fine. I like being able to add tests too :D

Reply:
> I like being able to add tests too :D

I refactored the code a bit to make it testable. Will add some tests today.